pilot/pkg/networking/core/v1alpha3/tracing.go

```diff
@@ -269,14 +270,14 @@ func zipkinConfigGen(hostname, cluster string) (*anypb.Any, error) {
 		TraceId_128Bit:           true,
 		SharedSpanContext:        wrapperspb.Bool(false),
 	}
-	return anypb.New(zc)
+	return protoconv.MessageToAny(zc), nil
```

Comment:
doesn't have to be in this PR, but the repeated `protoconv.MessageToAny(), nil` in these `*Gen` fns makes me wonder if `typedConfigGenFromClusterFn` should be refactored to drop the `error` return parameter (as nothing seems to use it).